### PR TITLE
Generalized mongo DB config

### PR DESCRIPTION
**Pull request by AnsahMohammad**
Added MONGO_DB configuration patch

### CODE REVIEW

**Comment by AnsahMohammad** [comment]
SO from now-on the Mongo gets intialized with custom URI (if defined) or else go with the default URL

File: server.py [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
 cors = CORS(app)
 
 
-gpt_neo_key = os.environ.get("GPTNEO")
-mongo_pass = os.environ.get("MONGO")
-
-mongo_uri = "mongodb+srv://mohdansah10:" + mongo_pass + "@cluster0.dsdb23w.mongodb.net/"
+gpt_neo_key = os.environ.get("GPTNEO", None)
+mongo_uri = os.environ.get("MONGO", "mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/")
 headers = {"Authorization": gpt_neo_key}
```
Comment (gobbledy-gook) [inline server.py:26]: can removes this line too

Comment (AnsahMohammad) [reply server.py:26]: You want to remove the line defining the URL ?

Comment (AnsahMohammad) [reply server.py:26]: Oh okay, let me try

Comment (gobbledy-gook) [reply server.py:26]: alright works with that line also..

**Review by gobbledy-gook — APPROVED** [review]

**Pull request merged**